tests/integration/src/main/groovy/com/datadog/appsec/php/docker/AppSecContainer.groovy

```diff
@@ -117,6 +117,33 @@ class AppSecContainer<SELF extends AppSecContainer<SELF>> extends GenericContain
         trace
     }
 
+    @CompileStatic(TypeCheckingMode.SKIP)
+    Object tracesFromRequest(String uri,
```

Comment:
Not the end of the world but I would change `traceFromRequest` to use this and pick the first one instead of having almost the same logic

Reply:
In an ideal world we wouldn't need this at all, which is what we'll have to figure out and then we'll remove this altogether.